### DISCUSSION TITLE
v/s knop adjustment

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#3894] [General]
Hi simmers,

anybody else who can´t use the v/s knop anymore after update to 1.14.5?

Greets Klaus

**Comment by Paytheon** [comment]
The V/S only allows me to select up to 1500 feet now. Also the F-PLN PERF stays in takeoff mode and never advances to climb. The AP in the FBW is useless now since the MSFS release.

**Comment by tracernz** [comment]
Fixed in https://github.com/flybywiresim/a32nx/pull/3881

**Reply by CptBayer** [reply]
bäääm .... yeah, workong again ... thanx a lot !!
